### PR TITLE
Switch to SRPMs in Copr for all projects

### LINKED ISSUE #1691: Switch to SRPMs in Copr for all projects
* [x] Remove the current condition to run SRPMs in Copr only when `srpm_build_deps` is present in the config.
* [x] Change the related tests (note: the code for building SRPMs is still needed in Koji builds)
* [x] Wait with this card for some time (we may receive some feedback for SRPMs in Copr from projects that started using it).

### PR DESCRIPTION
**Pull request by jpopelka**
TODO:

- [x] Write new tests or update the old ones to cover the new functionality.
- [ ] Update or write new documentation in `packit/packit.dev`.

Fixes #1691

---

RELEASE NOTES BEGIN
SRPMs for projects that installed the GitHub App before September 6th or don't have `srpm_build_deps` defined are now built in Copr as well.
RELEASE NOTES END

### CODE REVIEW

**Comment by jpopelka** [comment]
The tests should be fixed with packit/deployment#426

File: packit_service/worker/handlers/copr.py [hunk 1]
```diff
@@ -119,14 +117,7 @@ def get_packit_github_installation_time(self) -> Optional[datetime]:
         return None
 
     def run(self) -> TaskResults:
-        installed_at = self.get_packit_github_installation_time()
```
Comment (lbarcziova) [inline packit_service/worker/handlers/copr.py:122]: can we remove this method and the DATE_OF_DEFAULT_SRPM_BUILD_IN_COPR constant as well?

**Review by lbarcziova — APPROVED** [review]
:tada:

**Comment by TomasTomecek** [comment]
glad folks will still have a week to fix their packit.yamls

**Pull request merged**